src/components/Center/docs.md

```diff
@@ -0,0 +1,67 @@
+`<Center />` defines a max-width and ensures a horizontal padding.
+
+`<Breakout />` allows to adjust `size` (and positioning) of its content:
+- `regular` (default)
+- `narrow`
+- `tiny`
+- `breakout`
+- `breakoutLeft`
+- `float`
+- `floatSmall`
+- `floatTiny`
+
+```react
+<Center style={{backgroundColor: 'red'}}>
+  <Breakout size={'regular'} style={{backgroundColor: 'darkgreen', height: 20}}>
+  </Breakout>
+</Center>
+```
+
+```react
+<Center style={{backgroundColor: 'red'}}>
+  <Breakout size={'narrow'} style={{backgroundColor: 'darkgreen', height: 20}}>
+  </Breakout>
+</Center>
+```
+
+```react
+<Center style={{backgroundColor: 'red'}}>
+  <Breakout size={'tiny'} style={{backgroundColor: 'darkgreen', height: 20}}>
+  </Breakout>
+</Center>
+```
+
+```react
```

Comment:
use the [`responsive` catalog feature](https://docs.catalog.style/specimens/react#responsive-display) here to demonstrate that it goes away on mobile. And some stay? Like tiny?

Feel free to add an additional size to `responsiveSizes` (and give them better names) in `src/index.js` and then pass an array with the large size as default first option – `responsive: ['Desktop', 'Klein']`.

Reply:
if you don't use the responsive widget for some use `frame` to ensure the media query doesn't apply to the whole catalog page

Reply:
Done.